kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/KroxyliciousOperator.java

```diff
@@ -48,6 +60,29 @@ private InstallationMethod getInstallationMethod() {
         if (Environment.INSTALL_TYPE != InstallType.Yaml) {
             throw new UnsupportedInstallationType("Installation type " + Environment.INSTALL_TYPE + " not supported");
         }
-        return new KroxyliciousOperatorBundleInstaller(installationNamespace);
+        return new KroxyliciousOperatorBundleInstaller(installationNamespace, replicas);
+    }
+
+    /**
+     * Gets number of replicas.
+     *
+     * @return the number of replicas
+     */
+    public int getNumberOfReplicas() {
+        LOGGER.info("Getting number of replicas..");
+        return kubeClient().getDeployment(installationNamespace, Constants.KROXYLICIOUS_OPERATOR_DEPLOYMENT_NAME).getStatus().getReplicas();
+    }
+
+    /**
+     * Scale replicas to.
+     *
+     * @param scaledTo the number of replicas to scale up/down
+     * @param timeout the timeout
+     */
+    public void scaleReplicasTo(int scaledTo, Duration timeout) {
```

Comment:
This doesn't make sense.   You don't scale the operator (it will eventually be capable of running in a active/passive configuration for high availability but that's a different thing).

You will scale the operand.  That will be done via the KroxyProxy CR.  That's not implemented yet.

Reply:
That was a question that came to my mind. So, should we maintain those tests with manual deployments until we have the scalability for KroxyProxy CR?

Reply:
> should we maintain those tests with manual deployments 

No, I wouldn't bother.  That's going to lead to work that we throw away.

I would just mark the tests with the `@Disabled` annotation.

Enabling on the ability to scale the operand will actually be really quick.